### ISSUE TITLE
Font paths compiling to "/fonts/..."

### ISSUE THREAD

**Issue by neshmi** [mkhairi/materialize-sass#105]
The latest update to 0.97.7 seems to have somehow broken the font paths. I forked and changed from:

```
$roboto-font-path: "../fonts/roboto/" !default;
```

to 

```
$roboto-font-path: "/assets/roboto/" !default;
```

and now it works, but it seems that the initial should be correct. 

I have tried precompiling assets, etc. and if I rollback to 0.97.6 everything works. I've looked at the changes and can't see anything that would be breaking it. So... I'm confused.

I'm on Rails 5 FWIW.

**Comment by tfolk** [comment]
I was having the same issue. This fixed it for me:

$roboto-font-path: "roboto/" !default;
@import "materialize/components/roboto";
@import "materialize";

**Comment by mkhairi** [comment]
@neshmi, thanks for reporting..  fixed 6c7e191c0ec29ba75a8cc297cae6c26ceabdfbe1

**Comment by geovanygameros** [comment]
I was breaking my mind for this line. Thanks man!
